tests/data_science/SMSModel/test_stacking_holdout_comparison.py

```
@@ -0,0 +1,325 @@
+    monkeypatch: pytest.MonkeyPatch,
+    tmp_path: Path,
+) -> None:
+    """운영 artifact 기본 경로는 metadata 없이 로드되지 않는지 검증"""
+    model_path = tmp_path / "model.joblib"
+    joblib.dump(
+        {"classifier": DummyClassifier(["normal"], [0.1])},
+        model_path,
+    )
+    monkeypatch.setattr(comparison, "DEFAULT_MODEL_PATHS", (model_path,))
+
+    with pytest.raises(ValueError, match="metadata.json is required"):
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)